src/WebJobs.Script/Workers/Http/Configuration/HttpWorkerOptions.cs

```diff
@@ -15,8 +15,16 @@ public class HttpWorkerOptions
 
         public int Port { get; set; }
 
+        /// <summary>
+        /// Setting that enables the host to rebuild the initial invocation HTTP Request and send the copy to the worker process.
```

Comment:
Are you not seeing [SA1623 ](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1623.md)warning in VS? VS should also offer a context menu option to fix it for you.

Reply:
Yeah please switch to the correct docs for this "Gets or sets _"